vite.config.ts

```diff
@@ -44,9 +44,13 @@ export default defineConfig({
     Inspect(),
   ],
   test: {
```

Comment:
With the setup in this PR, 0 tests are run after `pnpm test:ui`.

Tests run using the following config, which also excludes `playground`, `docs` and `**/sponsorkit**/**` from coverage:

```
import { coverageConfigDefaults } from 'vitest/config'
```

```
  test: {
    environment: 'jsdom',
    globals: true,
    coverage: {
      provider: 'v8',
      exclude: [
        ...coverageConfigDefaults.exclude,
        'playground/**',
        'docs/**',
        '**/sponsorkit**/**',
      ],
    },
  },
```

Taken from here: [Vitest coverage.exclude docs](https://vitest.dev/config/#coverage-exclude)

Reply:
Hi @andretchen0 for me they work, but I updated the config with your suggestion